utils/R/do.conversions.R

```diff
@@ -1,9 +1,8 @@
-##' @export
+##' @export do.conversions
```

Comment:
do we really need to name do.conversions explicitly in the export?  If this was giving S3 issues then we should just change the name to do_conversions.